scmrepo/git/__init__.py

```diff
@@ -397,3 +397,17 @@ def stash_workspace(self, **kwargs):
 
     def _reset(self) -> None:
         self.backends.reset_all()
+
+    def describe(
+        self,
+        rev: str,
+        base: Optional[str] = None,
+        match: Optional[str] = None,
+        exclude: Optional[str] = None,
+    ) -> Optional[str]:
+        if base == "refs/heads" and self.get_rev() == rev:
+            try:
+                return base + "/" + self.active_branch()
```

Comment:
One slightly unrelated issue here is that `active_branch` is currently only implemented in the git-python backend, and ideally we should avoid adding new code in `scmrepo` that depends on git-python.

Really, `active_branch` is also probably a function that shouldn't be implemented via the backends at all (since it can be done at the `Git` level using the generic `get_ref`)